lib/stream/http.go

```diff
@@ -50,6 +50,8 @@ func (h *HTTP) Produce() (io.Reader, io.Reader, error) {
 
 	var err error
 
+	// We close the response body in Close().
+	// nolint[bodyclose]
```

Comment:
Oh!  Disregard my comment on the other PR, and +1 for the explanatory comment :-)